src/components/include/utils/byte_order.h

```diff
@@ -35,12 +35,20 @@
 
 #ifdef __QNX__
 #include <gulliver.h>
-#define BE_TO_LE32(x) ENDIAN_SWAP32(&(x));
-#define LE_TO_BE32(x) ENDIAN_SWAP32(&(x));
+#include <iostream>
+/*
+void ENDIAN_SWAP32( uint32_t * num );
+The ENDIAN_SWAP32() macro endian-swaps the value pointed to by num
+in place. Therefore, copying is necessary so as not to damage the
+original data.
+*/
```

Comment:
Please delver it as a separate commit

Reply:
@LuxoftAKutsan  Done in https://github.com/Ford-Luxoft/sdl_core/pull/30/commits/e8a0ccf961329f78b8528fcafb9172f71a013234